RackApp/app.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'rack'
+require 'active_support/time'
+
+class MyController
+  attr_reader :person, :time
+
+  def initialize(user, time)
+    @person = user
+    @time = time
+  end
+
+  def file(format)
+    File.read(File.join(__dir__, "template.#{format}.erb"))
```

Comment:
you can take it a bit forward and check if the file actually exists.. if not, we can throw some meaningful error.

do you know how to do it? or should I help you for a bit? :)